tasks/main.yml

```diff
@@ -1,27 +1,90 @@
 ---
 # tasks file for ansible-role-gridinit
+
+- include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+  tags: install
+
+- name: Install repository
+  include_tasks: "openio_install_repo_{{ ansible_os_family }}.yml"
```

Comment:
Isn't the ansible-role-repo-openio-sds suitable ?

Reply:
the next line allows it ;)

Reply:
Yes but why duplicating this over all roles ? Especially since we have a role to do it ?
What's the intent ?

Reply:
I do not know.
The other role is mainly there for credentials purposes no?
Do you prefer that I pass the boolean to `false` in the `default/main.yml`?

Reply:
Please use the appropriate role here. Would be a mess in case we change the repositories.